test/test_wait.py

```diff
@@ -0,0 +1,90 @@
+from opensquirrel import Circuit, CircuitBuilder
+from opensquirrel.ir import Bit
+
+
+def test_wait() -> None:
+    qc = Circuit.from_string(
```

Comment:
Same comments from `init`:
- Remove extra instructions, keep tests just to the minimum needed.
- Use parametrized tests.